new-pipeline/tutorial.html

```diff
@@ -13,21 +13,80 @@
     <link rel="stylesheet" href="style/jquery.tocify.css">
```

Comment:
Are changes to the tutorial necessary?

Reply:
These are from earlier PR, I can't understand why they are still showing up...